src/zenml/enums.py

```diff
@@ -291,3 +291,25 @@ class ModelStages(StrEnum):
     STAGING = "staging"
     PRODUCTION = "production"
     ARCHIVED = "archived"
+
+
+class ColorVariants(Enum):
+    """All possible color variants for frontend."""
+
+    GREY = 1
+    PURPLE = 2
+    RED = 3
+    GREEN = 4
+    YELLOW = 5
+    ORANGE = 6
+    LIME = 7
+    TEAL = 8
+    TURQUOISE = 9
+    MAGENTA = 10
+    BLUE = 11
+
+
+class TaggableResourceTypes(Enum):
+    """All possible resource types for tagging."""
+
+    MODEL = 1
```

Comment:
any reason we aren't using StrEnum here? it would make it more readable in the DB

Reply:
Save space - the only reason. 4 bytes vs 5+ (for non-unicode db, otherwise 10+)

Reply:
Do you think it is not worth saving?

Reply:
+1 to using StrEnum for more readable DB entries

Reply:
Few bytes per link is practically irrelevant compared to other stuff we dump into DB atm 😅

Reply:
This makes me feel pain as a former DB developer back in the day, but ok, let's go like this!

Reply:
Done